### PR TITLE
git-alias: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

File: pages/common/git-alias.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+> Create shortcuts for Git commands.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-alias>.
+
+- List all aliases:
+
+`git alias`
+
+- Make a new alias for Git:
```
Comment (bl-ue) [inline pages/common/git-alias.md:11]: Too wordy.  Proposal:
```suggestion
- Create a new alias:
```

File: pages/common/git-alias.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+`git {{alias}} "{{command}}"`
+
+- Search for existing alias:
```
Comment (bl-ue) [inline pages/common/git-alias.md:15]:
```suggestion
- Search for an existing alias:
```

**Comment by bl-ue** [comment]
>Didn't save (68e75b18f2838ba5247f14613b5b163bc6448a70)

So aggravating, isn't it? 😝

**Review by bl-ue — APPROVED** [review]

File: pages/common/git-alias.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-alias>.
+
+- List all aliases:
+
+`git alias`
+
+- Create a new alias:
+
+`git {{alias}} "{{command}}"`
```
Comment (bl-ue) [inline pages/common/git-alias.md:13]:
```suggestion
`git alias "name" "{{command}}"`
```

Comment (CleanMachine1) [reply pages/common/git-alias.md:13]: good find

**Comment by CleanMachine1** [comment]
Thats a lot of people... Didn't think I had done that many

**Review by marchersimon — APPROVED** [review]
Thanks @CleanMachine1

**Comment by marchersimon** [comment]
> Thats a lot of people... Didn't think I had done that many

I think if we merge now that shouldn't be a problem, since they probably won't look into it after it's merged.
